### PR TITLE
[WIP] Update to monitoring doc ... Telegraf Custom Config

### PR DESCRIPTION
**Pull request by mqarty**
# Description
Update the custom monitoring section to remove the *beta* tag and ensure customers understand how to use the new capabiltiy.

# Reasons
https://circleci.atlassian.net/browse/CIRCLE-12738

### CODE REVIEW

File: jekyll/_cci2/monitoring.md [hunk 1]
```diff
@@ -75,19 +75,25 @@ Enter your DataDog API Key.
...
+Custom Metrics via Telegraf configuration file is an alternative to the predefined CloudWatch and Datadog metric sections above. It can also be used instead of those sections for more fine grained control.
 
 ##### Configuration
 
 Configuration options are based on Telegraf's documented output plugins. See their documentation [here](https://github.com/influxdata/telegraf#output-plugins).
 
-For example, if you would like to use the InfluxDB Output Plugin the configuration would look like the following.
-
-![Custom](  {{ site.baseurl }}/assets/img/docs/metrics_custom2.png)
+For example, if you would like to use the InfluxDB Output Plugin you would need to follow these steps;
```
Comment (michelle-luna) [inline jekyll/_cci2/monitoring.md]: I don't see a reference to the new image in this PR?

File: jekyll/_cci2/monitoring.md [hunk 2]
```diff
@@ -75,19 +75,25 @@ Enter your DataDog API Key.
...
+1. SSH into the Servics Machine
+2. cd `/etc/circleconfig/telegraf/influxdb.conf
```
Comment (michelle-luna) [inline jekyll/_cci2/monitoring.md]: need a backtick after the influxbd.conf` for formatting

File: jekyll/_cci2/monitoring.md [hunk 3]
```diff
@@ -75,19 +75,25 @@ Enter your DataDog API Key.
...
+3. Adding the desired ouputs, e.g. ```
```
Comment (michelle-luna) [inline jekyll/_cci2/monitoring.md]: change e.g. to 'for example' and follwo it with a line break for correct formatting of the code block

File: jekyll/_cci2/monitoring.md [hunk 4]
```diff
@@ -75,19 +75,25 @@ Enter your DataDog API Key.
...
+[[output.influxdb]]
+  url = "http://52.67.66.155:8086"
+  database = "testdb"
+'''
+4. `docker restart telegraf` to restart the container to load or reload any changes.
```
Comment (michelle-luna) [inline jekyll/_cci2/monitoring.md]: Add 'Run' to the beginning of step 4 for clarity

**Comment by polymeris** [comment]
The content looks good to me.

**Comment by mqarty** [comment]
@michelle-luna updated based on your comments.

**Pull request merged**